plugins/crowdin/access_token.go

```diff
@@ -0,0 +1,63 @@
+package crowdin
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func AccessToken() schema.CredentialType {
+	return schema.CredentialType{
+		Name:    credname.AccessToken,
+		DocsURL: sdk.URL("https://developer.crowdin.com/configuration-file/#api-credentials-from-environment-variables"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.Token,
+				MarkdownDescription: "Token used to authenticate to Crowdin.",
+				Secret:              true,
+				Composition: &schema.ValueComposition{
+					Length: 80,
+					Charset: schema.Charset{
+						Lowercase: true,
+						Digits:    true,
+					},
+				},
+			},
+			{
+				Name:                fieldname.ProjectID,
+				MarkdownDescription: "Project ID used to authenticate to Crowdin.",
+				Optional:            true,
+				Composition: &schema.ValueComposition{
+					Charset: schema.Charset{
+						Digits: true,
+					},
+				},
+			},
+			{
+				Name:                fieldname.HostAddress,
+				MarkdownDescription: "Base URL (for Crowdin Enterprise)",
+				Optional:            true,
+				Composition: &schema.ValueComposition{
+					Charset: schema.Charset{
+						Lowercase: true,
+						Uppercase: true,
+						Digits:    true,
+						Symbols:   true,
+					},
+					Prefix: "https://",
+				},
+			},
+		},
+		DefaultProvisioner: provision.EnvVars(defaultEnvVarMapping),
+		Importer:           importer.TryEnvVarPair(defaultEnvVarMapping),
```

Comment:
@JoeKarow I am not familiar with Crowdin but based on initial research and testing, Crowdin configuration file can live in any folder, yes? Is that why we are not importing the credentials from the config file?

Reply:
Yes, that's correct. The `crowdin.yml` file lives in the root of each individual project. As far as I'm aware, there isn't any sort of system-wide config file.

Reply:
One problem that remains is, because the secrets live on the project-level config file, that takes precedence over environment variables:

https://developer.crowdin.com/configuration-file/#api-credentials-from-environment-variables

<img width="546" alt="image" src="https://github.com/1Password/shell-plugins/assets/18581859/54dfc63d-612b-4b0e-a049-f38a2e513946">

That's not ideal, and there's no functionality within Shell Plugins to remove/update config files to remove those secrets. 

I'll followup here to discuss how we can tackle this!

Reply:
Yeah, I saw that, too. That is what inspired the last sentence of the "Changelog" section of the PR.
Let me know if you have any other suggestions.

Reply:
@JoeKarow Ah, I see it now, thank you! We talked about this internally and here's the direction we'd like to take for now:

- Move forward with the current state of the PR.
- Document in the accompanying developer page, with instructions on how users should remove the secrets from the configuration files.
- In the future, think about automating this in the Shell Plugins codebase.